### DISCUSSION TITLE
Just wanted to say thanks

### DISCUSSION THREAD

**Discussion by ForDefault** [sfx101/docker-stacks#152] [General]
I have spent a long time trying to understand docker (I know, its simple... lol). But the fact I could get this working with just your app and get it running is extremely impressive to me. Best docker interface I have seen, by far, with almost no hassle. Would recommend to anyone learning or using docker since you have really outlined a working model of what is involved. Destroys docker desktop.

**Comment by sfx101** [comment]
Hey there, thank you so much for taking the time to write such positive feedback. Considering the time we spent developing DECK, it brings us immense joy. I hope the journey with Docker & DECK goes well for you! :)
